ur_robot_driver/src/hardware_interface.cpp

```diff
@@ -1292,7 +1297,7 @@ void HardwareInterface::startJointInterpolation(const hardware_interface::JointT
 {
   size_t point_number = trajectory.trajectory.points.size();
   ROS_DEBUG("Starting joint-based trajectory forward");
-  ur_driver_->writeTrajectoryControlMessage(urcl::control::TrajectoryControlMessage::TRAJECTORY_START, point_number);
+  ur_driver_->writeTrajectoryControlMessage(urcl::control::TrajectoryControlMessage::TRAJECTORY_START, point_number, robot_receive_timeout_);
```

Comment:
For trajectory control messages the default value is [a lot higher](https://github.com/UniversalRobots/Universal_Robots_Client_Library/blob/96351221eed78a040f5c2f2961b9ce46dfad9060/include/ur_client_library/ur/ur_driver.h#L560) actually, since we can tolerate missing communication for a longer period. Hence, I would suggest not adding it here.
```suggestion
  ur_driver_->writeTrajectoryControlMessage(urcl::control::TrajectoryControlMessage::TRAJECTORY_START, point_number);
```